tests/functional/adapter/warehouse_per_model/test_warehouse_per_model.py

```diff
@@ -24,35 +26,45 @@ def models(self):
             "special": d,
         }
 
+    @pytest.fixture(scope="class")
+    def profile_dir(self, profiles_root):
+        return str(profiles_root)
+
 
 class BaseSpecifyingCompute(BaseWarehousePerModel):
     """Base class for testing various ways to specify a warehouse."""
 
-    def test_wpm(self, project):
-        util.run_dbt(["seed"])
+    def test_wpm(self, project, profile_dir):
+        util.run_dbt(["seed", "--profiles-dir", profile_dir])
         models = project.test_config.get("model_names")
         for model_name in models:
             # Since the profile doesn't define a compute resource named 'alternate_warehouse'
             # we should fail with an error if the warehouse specified for the model is
             # correctly handled.
-            res = util.run_dbt(["run", "--select", model_name], expect_pass=False)
+            res = util.run_dbt(
+                ["run", "--select", model_name, "--profiles-dir", profile_dir],
+                expect_pass=False,
+            )
             msg = res.results[0].message
             assert "Compute resource alternate_warehouse does not exist" in msg
             assert model_name in msg
 
 
+@pytest.mark.skip("May fail non-deterministically due to issue in dbt test framework.")
```

Comment:
Shouldn't we be disable TestWarehousePerModel and leaving these ones to run.  I though we should be disabling the tests that use the profiles_config_update fixture.

Reply:
I feel like it's more valuable to validate the affirmative case, and even with the bug we identified, these additions to the profile shouldn't impact any other tests (since their schema's won't reference the compute block)